pylint/utils.py

```diff
@@ -1145,3 +1145,19 @@ def _rest_format_section(stream, section, options, encoding=None, doc=None):
             value = _encode(_format_option_value(optdict, value), encoding)
             print(file=stream)
             print('  Default: ``%s``' % value.replace("`` ", "```` ``"), file=stream)
+
+
+def get_duplicated(items):
+    """Get duplicated items
+    :param iter items: Items to extract duplicated
+    :returns: Duplicated extracted from items
+    :rtype: list
+    """
+    unique_items = set()
+    duplicated_items = set()
```

Comment:
You can use a collections.Counter here. This might look as (untested):

c = collections.Counter(items)
return [key for key, value in c.items() if value == 1]